app/decorators/user_decorator.rb

```diff
@@ -100,11 +100,15 @@ def verified_account_partial
 
   private
 
+  def model
+    __getobj__
```

Comment:
I'm not used to seeing SimpleDelegator, so I think this is a little sneaky?

What if we just made a plan-old-Ruby-class with a simple contructor?

```ruby
class UserDecorator
  def initialize(model)
    @model = model
  end
  # ...
end
```